### ISSUE TITLE
Error youtube-dl Encoder

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo-Encoder#532]
@DanielnetoDotCom Danny, I have this error message in Encoder, referring to youtube-dl.
![image](https://github.com/WWBN/AVideo-Encoder/assets/37880073/037709cd-02fd-4be1-b0d5-bacd5f63582a)

> ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see https://yt-dl.org/update on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

I tried accessing the URL https://yt-dl.org/update
But apparently it was legally closed.

youtube-dl is installed here:

```
ubuntu@ubuntu-server-arm-guibla:~$ sudo youtube-dl
Usage: youtube-dl [OPTIONS] URL [URL...]

youtube-dl: error: You must provide at least one URL.
Type youtube-dl --help to see a list of all options.

```

Public encoder works normally...

![image](https://github.com/WWBN/AVideo-Encoder/assets/37880073/72c3fcb4-7dd0-4844-ad7f-f74a126b5822)

Can you tell me what this could be?

**Comment by DanielnetoDotCom** [comment]
Install this

``````
sudo curl -L https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp -o /usr/local/bin/yt-dlp && sudo chmod a+rx /usr/local/bin/yt-dlp 
``````

**Comment by gbrn** [comment]
Danny, I checked that you were running an earlier version. This is because the official website is banned.
![image](https://github.com/WWBN/AVideo-Encoder/assets/37880073/196c144e-91d5-4333-bb8d-51bc7fe1e0e8)

I would already ask you about the possibility of switching to yt-dlp.
I installed it here and it worked. OK

I think it is pertinent to update the installation WIKI:
https://github.com/WWBN/AVideo-Encoder/wiki
